packages/language-service/lib/plugins/vue-autoinsert-dotvalue.ts

```diff
@@ -19,6 +20,12 @@ export function create(ts: typeof import('typescript')): ServicePlugin {
 					if (!isCharacterTyping(document, lastChange))
 						return;
 
+					const req = ++currentReq;
+					// Wait for tsserver to sync
+					await sleep(250);
+					if (req !== currentReq)
+						return;
+
 					const enabled = await context.env.getConfiguration?.<boolean>('vue.autoInsert.dotValue') ?? true;
 					if (!enabled)
 						return;
```

Comment:
Wouldn't it make sense to check this configuration before the newly added code?

Reply:
getConfiguration has a send request cost before Volar has cached the option, so it is best to place it after sleep.

Reply:
But now the unnecessary sleep will trigger for every document change, even if one is not using this option. Seems more wasteful than getting a configuration once that gets cached later.